src/iotjs_binding.c

```diff
@@ -167,7 +167,7 @@ void iotjs_jval_destroy(iotjs_jval_t* jval) {
 
 
 static void iotjs_jval_destroy_norelease(iotjs_jval_t* jval) {
-  IOTJS_VALIDATED_STRUCT_DESTRUCTOR(iotjs_jval_t, jval);
+  IOTJS_VALIDATED_STRUCT_DESTRUCTOR_VALIDATE(iotjs_jval_t, jval);
```

Comment:
Suggested name ``IOTJS_VALIDATED_STRUCT_DESTRUCTOR_VALIDATE gives more information. But it is too long and some word are duplicated.

Basically, I don't like current prefix  ``IOTJS_VALIDATED_STRUCT``:
1) Too long. 
2) Also this structure does not validated, but validatable. But it seems validatable is not popular word. Sanity checkable would be better. But it is too long.

I cannot suggest better candidate that satisfy aboves.
What about landing unused variable warning only? And rename the name in another PR?